pkgs/development/libraries/taco/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, fetchgit
+, cmake
+, enablePython ? false
+, python ? null
+}:
+
+let pyEnv = python.withPackages (p: with p; [ numpy scipy ]);
+
+in stdenv.mkDerivation rec {
+  pname = "taco";
+  version = "unstable-2022-08-02";
+
+  src = fetchgit {
+    url = "https://github.com/tensor-compiler/${pname}.git";
+    rev = "2b8ece4c230a5f0f0a74bc6f48e28edfb6c1c95e";
+    fetchSubmodules = true;
+    hash = "sha256-PnBocyRLiLALuVS3Gkt/yJeslCMKyK4zdsBI8BFaTSg=";
+  };
+
+  # Remove test cases from cmake build as they violate modern C++ expectations
+  patches = [ ./taco.patch ];
+
+  nativeBuildInputs = [ cmake ];
+
```

```suggestion
 buildInputs = lib.optional stdenv.isDarwin llvmPackages.openmp;
```
(or simply disable darwin if it does not work).

---

I've added it.